src/PKSim.Infrastructure/InfrastructureRegister.cs

```diff
@@ -124,25 +129,50 @@ public static void RegisterSerializationDependencies()
          container.Register(typeof(IXmlReader<>), typeof(XmlReader<>));
          container.Register(typeof(IXmlWriter<>), typeof(XmlWriter<>));
 
-         //load repository to trigger initialization
+         //Core serialization to serialize to pkml compatible with core
+         container.AddRegister(x => x.FromType<CoreSerializerRegister>());
+      }
+
+      public static void LoadDefaultEntities(IContainer container)
```

Comment:
I had to reoranize all this stuff to make sure that I could change the path of the PKSim DB dynamically. SOmehow, the logic to load the db locally does not work because the path returned is the path from rClr. Anyways...it took me a whule to get everything working again but this is now ok